CONTRIBUTING.md

```diff
@@ -0,0 +1,205 @@
+git pull upstream master
+```
+
+This will update the local copy of the Rasa repository to the latest version.
+
+#### 4. Implement your code contribution
+
+At this point, you are good to make changes to the files in the local directory of your project.
+
+Alternatively, you can create a new branch which will contain the implementation of your contribution. To do that, run:
+
+```
+git checkout -b name-of-your-new-branch
+```
+
+#### 5. Push changes to your forked repository on GitHub
+
+Once you are happy with the changes you made in the local files, push them to the forked repository on GitHub. To do that, run the following commands:
+
+```
+git add .
+git commit -m ‘fixed a bug’
+git push origin name-of-your-new-branch
+```
+
+This will create a new branch on your forked Rasa repository, and now you’re ready to create a Pull Request with your proposed changes!
+
+#### 6. Opening the Pull Request on Rasa Open Source
+
+Head to the forked repository and click on a _Compare & pull_ request button.
+
+![](https://www.rasa.com/assets/img/contributor-guidelines/openpr-1.png)
+
+This will open a window where you can choose the repository and branch you would like to propose your changes to, as well as specific details of your contribution. In the top panel menu choose the following details:
+
+- Base repository: RasaHQ/rasa
+- Base branch: master
+- Head repository: your_github_username/rasa
+- Head branch: name-of-your-new-branch
+
+![](https://www.rasa.com/assets/img/contributor-guidelines/openpr-2.png)
+
+Next, make sure to update the pull request card with as many details about your contribution as possible. _Proposed changes_ section should contain the details of what has been fixed/implemented, and Status should reflect the status of your contributions.
+
+If you are ready to get feedback on your contribution from the Rasa team, tick the _made PR ready for code review_ box.
+
+Once you are happy with everything, click the _Create pull request_ button. This will create a Pull Request with your proposed changes.
+
+![](https://www.rasa.com/assets/img/contributor-guidelines/openpr-3.png)
+
+#### 7. Signing the Contributor Licence Agreement (CLA)
+
+To merge your contributions to the Rasa codebase, you will have to sign a Contributor License Agreement (CLA).
+
+It is necessary for us to know that you agree for your code to be included into the Rasa codebase and allow us to use it in our later releases. You can find a detailed Rasa Contributor Licence Agreement [here](https://cla-assistant.io/RasaHQ/rasa).
+
+#### 8. Merging your PR and the final steps of your contribution
+
+Once you sign the CLA, a member from the Rasa team will get in touch with you with the feedback on your contribution. In some cases, contributions are accepted right away, but often, you may be asked to make some edits/improvements. Don’t worry if you are asked to change something - it’s a completely normal part of software development.
```

Comment:
Also mention please do not request a review, a reviewer will be automatically assigned to you